src/normalized.css

```diff
@@ -0,0 +1,19 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html,
+input {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  color: var(--text-color);
+  font-weight: 300;
```

Comment:
Be consistent with your font-weight usage. If there is no specific design requirement for a lighter font-weight, it's common to use the default weight of 400 for readability.